AudioBar/AudioBar.swift

```diff
@@ -42,6 +42,7 @@ public struct AudioBar: Program {
             case setCurrentTime(TimeInterval)
         }
         case player(Player)
+        case playerDidPlayToEnd
```

Comment:
This is an action. It needs to be in imperative moon, e.g. `callDelegatePlayerDidPlayToEnd`, `informDelegateThatPlayerDidPlayToEnd`, etc.